sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPlugin.kt

```diff
@@ -12,6 +12,7 @@ interface LocationComponentPlugin :
   MapStyleObserverPlugin,
   LifecyclePlugin,
   ContextBinder,
+  LocationComponentAccuracyRadiusSettingsInterface,
```

Comment:
isn't that still a breaking change if user implements own version of LocationComponentPlugin?

Reply:
yes, but we should treat plugin interfaces as internal, with v11 we should put them into an internal package. This avoids the breaking change issue while still allowing developers to benefit from our plugin system.

Reply:
@tobrun I'm missing smth, I guess. You saying that `LocationComponentAccuracyRadiusSettingsInterface` will be internal? And the `LocationComponentPlugin` would still be public?

Reply:
Here I'm still don't understand why should the LocationComponentPlugin implement the `LocationComponentAccuracyRadiusSettingsInterface`, instead of LocationComponentPluginImpl implementing it, similarly to implementing `LocationIndicatorAccuracyRadiusHandler`

Reply:
@yunikkk this is your idea https://github.com/mapbox/mapbox-maps-android/pull/1016#discussion_r782999889

Reply:
Another idea is to introduce a `LocationComponentPlugin2` that extends the `LocationComponetPlugin` interface, where our `LocationComponentPluginImpl` will implement `LocationComponentPlugin2`?